### ISSUE TITLE
ModuleNotFoundError: No module named 'spiders'

### ISSUE THREAD

**Issue by xc21** [mfrodl/linkedin-scraper#1]
Hi Martin,

**Thanks a lot for sharing things. When I was running the project with Windows cmd, I got the error message for "ModuleNotFoundError: No module named 'spiders'" the system log is as following:**

(C:\Users\caok\AppData\Local\Continuum\anaconda3) C:\Users\caok>linkedin-scraper
 --output supply chain analyst.csv supply chain analyst
Traceback (most recent call last):
  File "C:\Users\caok\AppData\Local\Continuum\anaconda3\Scripts\linkedin-scraper
-script.py", line 11, in <module>
    load_entry_point('linkedin==0.1.0', 'console_scripts', 'linkedin-scraper')()

  File "C:\Users\caok\AppData\Local\Continuum\anaconda3\lib\site-packages\pkg_re
sources\__init__.py", line 570, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "C:\Users\caok\AppData\Local\Continuum\anaconda3\lib\site-packages\pkg_re
sources\__init__.py", line 2751, in load_entry_point
    return ep.load()
  File "C:\Users\caok\AppData\Local\Continuum\anaconda3\lib\site-packages\pkg_re
sources\__init__.py", line 2405, in load
    return self.resolve()
  File "C:\Users\caok\AppData\Local\Continuum\anaconda3\lib\site-packages\pkg_re
sources\__init__.py", line 2411, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "C:\Users\caok\AppData\Local\Continuum\anaconda3\lib\site-packages\linked
in-0.1.0-py3.6.egg\linkedin\__main__.py", line 8, in <module>
ModuleNotFoundError: No module named 'spiders'

**Do you have any idea why this is happening?

Thanks a lot!!**

**Comment by mfrodl** [comment]
Hello Xun,

thank you for opening this issue. As the scraper has only really been tested for Python 2.7 in a Linux environment, some bugs might have escaped my attention. This seems to be one of them.

I suspect the error is caused by attempting to do an implicit relative import in `__main__.py`. Specifically, this line:
```python
from spiders import LinkedinSpider
```
should be replaced with
```python
from linkedin.spiders import LinkedinSpider
```

I pushed the fix to branch `1` as commit 9d394b7d5299873baa413ae9d160d112da603567. Could you check it out, run the installation script once again and see what it does?

**Comment by xc21** [comment]
Thanks a lot, Martin! That helped resolve the issue on Windows 8 cmd with Python 3.6.
